homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="./css/main.css">
+        <link rel="stylesheet" type="text/css" href="./css/aboutStyle.css">
+        <title>About</title>
+    </head>
+    <body>
+        <header class="header">
+            <img class="header-logo" src="./Img/logo.svg" width="55px" height="57.323px">
+            <nav class="nav-menu">
+                <ul>
+                    <li>
+                        <a href="./games.html">Games</a>
+                    </li>
+                    <li>
+                        <a href="./index.html">About</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main class="main__section">
+            <img src="./Img/image.svg" width="532px" height="356px">
+            <div class="about-main__content">
+                <h1 class="about-me__header">About me</h1>
+                <div class="main__paragraph">
+                    <p class="main-content">
+                        Hi! My name is Vladyslav Brusentsov and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.<br>
+                        <br>
```

Comment:
This old practice of using tags like 'br', 'i', 'b' should be avoided. Please remove all 'br' tags and split the content by the <p> tag to create paragraphs in the layout